### DISCUSSION TITLE
Is there a way to disable the Desktop context menu item "New Portal and Folder"?

### DISCUSSION THREAD

**Discussion by just4747** [Ross-Patterson/Portals-Desktop-Organization#145] [Q&A] [answered]
I do not want this additional item added to my Desktop context menu, as it crowds it up and isn't necessary since you can create a Portal from the app itself. Is there a built in way to remove this without editing regedit? If not, can we please get the option to disable this?

**Comment by dking02** [comment]
You can easily disable/enable the “Create Portal from Folder” context menu item using _**ShellMenuView**_, a small portable program which is part of the NirSoft freeware utilities. It is available for download in 32-bit or 64-bit versions at: 

[https://www.nirsoft.net/utils/shell_menu_view.html](https://www.nirsoft.net/utils/shell_menu_view.html)

The download link is located towards the bottom of the page. 

Unzip the contents to a folder and run the program. The program will display a list of Explorer static context menu items sorted alphabetically.

1. Scroll down to the “Create Portal from Folder” item in column 1 (Menu Name)
2. Select the item (row) and right-click, and 
3. Select “Disable the selected items” (F7).

**Reply by just4747** [reply]
Understood - definitely aware of the multiple ways to remove it manually but is it possible you might add the option to turn it on/off right in Portals settings in future for convenience? Thanks!

**Comment by Ross-Patterson** [comment, marked as answer]
I agree that this should be optional. Its just a registry key, so I will add it to the installer like this in the next update:
![image](https://github.com/Ross-Patterson/Portals-Desktop-Organization/assets/66630787/db912b97-8a30-47e0-9117-7037066a66ec)

Note that a full uninstall is required to remove the registry keys, simply updating won't work.

**Reply by just4747** [reply]
Amazing - thanks so much for that!

**Comment by bandaid** [comment]
You can also add the option within Settings to disable or enable the option. Either log off, restart Explorer or reboot.
